code-reports/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Couldn't this be also done directly in the top-level pom with a profile ?

Or what other purpose does this module serve ?